contrib/prometheus-protobufs/README

```diff
@@ -0,0 +1,2 @@
+Files "prompb/remote.proto" and "prompb/types.proto" were downloaded from the Prometheus repository:
+https://github.com/prometheus/prometheus/tree/main/prompb
```

Comment:
The whole repository https://github.com/prometheus/prometheus/tree/main/prompb is quite big, and I need only two small files `remote.proto` and `types.proto` from it, so I decided not to add the prometheus repository as a submodule and just download these two files to our `contrib/prometheus-protobufs/` folder instead.